### PR TITLE
 Moved multisite screens markup to dedicated view files

### LINKED ISSUE #21: Separate views
Please, add separate views for the Logs and Default page in MS mode:
/wp-admin/network/admin.php?page=sg-cachepress
https://sgmstest-mapdomain.com/wp-admin/network/admin.php?page=sg-cachepress-log

### PR DESCRIPTION
**Pull request by Rarst**
Fixes #21

### CODE REVIEW

File: sg-cachepress.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ function sgcachepress_load_textdomain() {
 require plugin_dir_path( __FILE__ ) . 'class-sg-cachepress-ssl.php';
 require plugin_dir_path( __FILE__ ) . 'class-sg-cachepress-multisite.php';
 require plugin_dir_path( __FILE__ ) . 'class-sg-cachepress-log.php';
-//require plugin_dir_path( __FILE__ ) . 'class-sg-cachepress-performance-tool.php';
+require plugin_dir_path( __FILE__ ) . 'class-sg-cachepress-performance-tool.php';
```
Comment (GaryJones) [inline sg-cachepress.php]: The changes in this file seem unrelated to moving markup into view files?

File: views/network-settings.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+?>
+<div class="wrap">
+
+	<h1><?php esc_html_e( 'SG Optimizer', 'sg-cachepress' ); ?></h1>
+
+	<h2><?php esc_html_e( '.htaccess Options', 'sg-cachepress' ); ?></h2>
+
+	<p>
+		<a id="sg-cachepress-gzip-toggle" class="<?php echo $gzip_enabled ? 'toggleon' : 'toggleoff'; ?>" href="#"></a>
+		<?php esc_html_e( 'gZip Compression', 'sg-cachepress' ); ?>
```
Comment (GaryJones) [inline views/network-settings.php]: This should be `gzip` or `Gzip`, but not `gZip`.

Comment (Rarst) [reply views/network-settings.php]: Verbatim from provided design for the record.
